### PR TITLE
solana: fix math error

### PR DESCRIPTION
**Pull request by a5-pickle**
Settle auction none should have been factoring the init auction fee, which is compensation for posting the fast VAA.

### CODE REVIEW

**Review by gator-boi — APPROVED** [review]
clean

File: solana/programs/matching-engine/src/processor/auction/settle/none/mod.rs [hunk 1]
```diff
@@ -60,7 +60,9 @@ fn settle_none_and_prepare_fill(
     // Pay the `fee_recipient` the base fee. This ensures that the protocol relayer is paid for
```
Comment (johnsaigle) [inline solana/programs/matching-engine/src/processor/auction/settle/none/mod.rs:60]: Can you update the comment here? Something like:
```suggestion
    // Pay the `fee_recipient` the base fee plus the auction account creation fee. This ensures that the protocol relayer is paid for
```

**Review by johnsaigle — APPROVED** [review]
Looks good, added a note to fix a comment but could be done in another PR

File: solana/programs/matching-engine/src/processor/auction/settle/none/mod.rs (hunk 1 above)
Comment (a5-pickle) [reply solana/programs/matching-engine/src/processor/auction/settle/none/mod.rs]: I decided to reword the comment a bit more. Does this look all right? https://github.com/wormhole-foundation/example-liquidity-layer/pull/92/commits/a4ee863ce2ed4a2643150149382f29ee8d0c00a2

**Review by johnsaigle — APPROVED** [review]

**Pull request merged**